apps/api/src/persistence/database/arango-database.ts

```diff
@@ -30,9 +30,6 @@ const interpretQueryOperatorForAQL = (operator: QueryOperator): string => {
     return lookupResult;
 };
 
-/**
- * TODO It seems we should sanatize inputs here.
```

Comment:
Stale comment. We already did this and added test coverage for protection against AQL injection.